conf/messages.en

```diff
@@ -261,6 +261,10 @@ pageNotFound.p3         = If the web address is correct or you selected a link o
 pageNotFound.link       = HMRC Online Services Helpdesk
 pageNotFound.p4         = if you need to speak to someone.
 
+accessDenied.title      = Sorry, there is a problem with the service
+accessDenied.heading    = Sorry, there is a problem with the service
+accessDenied.link       = Return to the Construction Industry Scheme start page.
```

Comment:
I believe when a link is an entire sentence, it should not have a full-stop, please check with Moin/UCD team. 
If it should have a full stop, then the full stop should not be part of the link.